rails/app/models/user.rb

```diff
@@ -26,19 +26,14 @@ def self.find_for_token_authentication(conditions)
   NO_EMAIL_STRING = 'no-email-'
   NO_EMAIL_DOMAIN = 'concord.org'
 
-  has_many :investigations
-  has_many :activities
   has_many :interactives
-  has_many :sections
-  has_many :pages
   has_many :external_activities
   has_many :security_questions
 
   has_one :portal_teacher, :dependent => :destroy, :class_name => "Portal::Teacher", :inverse_of => :user
   has_one :portal_student, :dependent => :destroy, :class_name => "Portal::Student", :inverse_of => :user
 
   has_many :open_responses, :class_name => 'Embeddable::OpenResponse'
```

Comment:
Shouldn't the Open Responses be removed too?

Reply:
It should be, thanks. I've just done it in the last commit.